### ISSUE TITLE
v9.0c Screenwidth fix for MS Explorer 11

### ISSUE THREAD

**Issue by mgielissen** [odoo/odoo#12929]
Treeview, Form and Kanban in Odoo 9 community are not correctly displayed when using MS Explorer 11. The screen width is way to small or to large.

**Comment by mgielissen** [comment]
This will fix the issues with MS Explorer 11:

```
// Make formwidth relative to screen
.openerp .oe_form_sheet_width {
    max-width: 90%;
}

// Explorer 11 fix to use fullscreen

@media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {

	.oe-view-manager-view-kanban, .oe-view-manager-view-form, .oe-view-manager-view-list, .oe-view-manager-view-calendar, .oe-view-manager-view-pivot, .oe-view-manager-view-graph {
        width: 100% !important;
	}
}
```

**Comment by sle-odoo** [comment]
![bs_win7_ie_11 0](https://cloud.githubusercontent.com/assets/7601090/17177918/bb3fa326-5413-11e6-8653-1a38e4b2965b.jpg)
![bs_win7_ie_11 0 1](https://cloud.githubusercontent.com/assets/7601090/17177921/bf7445d2-5413-11e6-8607-2f6a51f46da2.jpg)

on the first screenshot, the width 100% is not respected
on the second, the form view is not centered

**Comment by mgielissen** [comment]
See my fix for Sales Dashboard: https://github.com/Openworx/themes/commit/a72cf8e7eb28f4b46c41a1cd252afbd7f2a40d11

**Comment by sle-odoo** [comment]
@mgielissen could you try my patch and confirm it fixes the issue at your end? Looks ok to me on ie11 and edge 14. Waiting for your feedback.

**Comment by mgielissen** [comment]
@sle-odoo Your patch fixe

s the screens. 

The User Preference screen doesn't display correctly.

![naamloos](https://cloud.githubusercontent.com/assets/7700396/17260249/98512372-55cf-11e6-9536-9e9462b3969f.png)

**Comment by sle-odoo** [comment]
@mgielissen could you try again with the last patch? thanks

**Comment by mgielissen** [comment]
User Preference Screen works fine in IE11.

**Comment by sle-odoo** [comment]
@mgielissen actually it is not: the save and discard button are not displayed correctly. however normal modals like the print general ledger are ok. It's a little tricky...

**Comment by sle-odoo** [comment]
fixed with rev 3e2faec034c7bc85066b22a00e70be6da8e60a24
